recipes/adcc/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = "0.15.10" %}
+
+package:
+  name: adcc
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/adcc/archive/v{{ version }}.tar.gz 
+  sha256: 8cd0fc53a0e4862421ff45b27058a7bae7e54738eea3a57ef75d221908f980a3
+
+build:
+  number: 0
+  skip: true  # [win or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - libtensorlight >=3.0.1
+    - pkgconfig
+    - wget
+  host:
+    - python
+    - setuptools
+    - pip
+    - libtensorlight >=3.0.1
+    - pybind11 >=2.6
+    # Testing:
+    - pytest
+    - pytest-cov
+    - pyyaml
+    - pandas >=0.25.0
+    # Standard dependencies:
+    - numpy >=1.14
+    - h5py >=2.9
+    - scipy >=1.2
+    - tqdm >=4.30
+    - opt_einsum >=3.0
+  run:
+    - python
+    - setuptools
+    - libtensorlight >=3.0.1
+    - numpy >=1.14
+    - h5py >=2.9
+    - scipy >=1.2
+    - tqdm >=4.30
+    - opt_einsum >=3.0
+
+test:
+  requires:
+    - pandas >=0.25.0
+  imports:
+    - adcc
+  commands:
+    - export PYLIB_EXT=`$PYTHON -c 'import sysconfig as s; print(s.get_config_var("EXT_SUFFIX"))'`
+    - test -f $PREFIX/lib/python{{ PY_VER }}/site-packages/libadcc$PYLIB_EXT
+
+about:
+    home: https://adc-connect.org
+    dev_url: https://github.com/adc-connect/adcc
+    doc_url: https://adc-connect.org
+    license: GPL-3.0-only
```

Seems like "or later" is the [right one](https://github.com/adc-connect/adcc/blob/8b03f3cf057f588262bf24b5d0c843dcfed4f7a2/adcc/AdcMatrix.py#L11-L12):
```suggestion
    license: GPL-3.0-or-later
```

---

Done.